data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/AuditErrorMessage.java

```diff
@@ -75,7 +75,7 @@ public static ILeafResourceId create() {
          * Starter method for the Builder class that uses the {@link AuditableAuthorisedDataRequest} objects for the
          * construction and will expect the next to builder method to be for the attribute map
            *
-         * @param auditableAuthorisedDataRequest   the client request received by the data-service
+         * @param auditableAuthorisedDataRequest   the client information request received by the Data Service
```

Comment:
I'm not sure this reads correctly. It makes it sound like the request param contains information on the client and not information about the request from client

Reply:
Short answer is both.  I'll re-word it to be clear.